tests/test_config_parsing/config_dict_generator.py

```diff
@@ -36,6 +35,14 @@ def directory_names(draw):
     return "dir" + draw(words)
 
 
+@st.composite
+def report_steps(draw):
+    rep_steps = draw(
+        st.lists(st.integers(min_value=0, max_value=100), min_size=4, unique=True)
+    )
+    return ",".join(str(step) for step in sorted(rep_steps))
+
```

Comment:
nice. i was a bit unsure if these should be sorted or not (or rather whether we catch whether they're sorted or not)